src/app/helper/mail-templates/testimonial_notification/mail.html

```diff
@@ -0,0 +1,335 @@
+<!doctype html>
```

Comment:
The design seems to be identical for every mail html file I've seen so far. Could you please extract the duplicated parts so that the templates just contain the text that differs?